components/scream/src/physics/p3/tests/p3_main_unit_tests.cpp

```diff
@@ -340,6 +340,7 @@ static void run_bfb_p3_main()
         {d.pres           , {1.00000000E+02 , 9.87111111E+04}},
         {d.dz             , {1.22776609E+02 , 3.49039167E+04}},
         {d.nc_nuceat_tend , {0              , 0}},
+        {d.nccn_prescribed, {0              , 0}},
```

Comment:
I think nccn_prescribed needs to be positive if do_prescribed_CCN is true. @hassanbeydoun - what's a good number here?